app/src/main/java/org/simple/clinic/registration/phone/RegistrationPhoneScreenController.kt

```diff
@@ -156,4 +158,13 @@ class RegistrationPhoneScreenController @Inject constructor(
       createdAt = loggedInUserPayload.createdAt,
       updatedAt = loggedInUserPayload.updatedAt
   )
+
+  private fun showLoggedOutOnThisDeviceDialog(events: Observable<UiEvent>): Observable<UiChange> {
+    return events
+        .ofType<RegistrationPhoneScreenCreated>()
+        .flatMap { userSession.isUserUnauthorized() }
+        .take(1)
```

Comment:
`take(1)` will complete the upstream. 
Can you change this to the following? 

`.flatMap { userSession.isUserUnauthorized().take(1) }`

Reply:
Whoops, I didn't know this. I'll fix this.